game-card.html

```diff
@@ -361,7 +362,7 @@
             }
 
             _computeFigureImage(rank, symbol) {
-                var rankName = GameCard.FIGURES.includes(rank.toLowerCase()) && GameCard.RANKS[rank.toLowerCase()],
+                var rankName = GameCard.FIGURES.indexOf(rank.toLowerCase()) != -1 && GameCard.RANKS[rank.toLowerCase()],
```

Comment:
@Artur- why did you move from #includes to #indexOf ? is this because it is out of the ES2015 sec ?